### DISCUSSION TITLE
How can I create element and event?

### DISCUSSION THREAD

**Discussion by hellpirat** [TheSpyder/rescript-webapi#14] [Q&A] [answered]
Playing around with rescript-webapi, stuck from the beginning.

I am using this repo - https://github.com/rescript-lang/rescript-project-template

Just copy-paste code from https://github.com/tinymce/rescript-webapi/blob/main/tests/Webapi/Dom/Webapi__Dom__EventTarget__test.re this file.

Get an error on 4th line:

```
 We've found a bug for you!
fun/rescript-project-template/src/Demo.res:4:14-21

  2 │ open EventTarget;
  3 │ 
  4 │ let target = document->Document.createElement("strong")->Element.asEvent
    │ Target;
  5 │ let event = Event.make("my-event");
  6 │ let handleClick = _ => print_endline("asd");

  This has type:
    Dom.document (defined as
      Dom.eventTarget_like<Dom._node<Dom._document<Dom._baseClass>>>)
  Somewhere wanted: string

FAILED: cannot make progress due to previous errors.
```

What I am doing wrong? :)

**Comment by TheSpyder** [comment, marked as answer]
Because we forgot to publish a release after implementing #6 🤦‍♂️ 

I'll make time to push up the new release soon. For now, if you change it to `document |> Document.createElement("strong")` it should work.

**Comment by TheSpyder** [comment]
v0.1.0 has now been published with the pipe first swap.

**Reply by hellpirat** [reply]
Feel like there is no change..

I tried to install it, reinstall and nothing changed 🤔 

```
npm install rescript-webapi
```

**Reply by hellpirat** [reply]
https://github.com/hellpirat/rescript-webapi-examles I made a repo to reproduce

**Reply by TheSpyder** [reply]
Your project still depends on `0.0.1`. Due to how NPM treats version numbers, `0.1.0` is a breaking change (this was by design, since we changed just about every API). To update you need to either edit the `package.json` to change the number or completely remove and re-add the dependency.

**Reply by hellpirat** [reply]
oh, lol. Thank you, my bad :D
